.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,16 @@
-# Overview
-<!-- What this PR does, and why is needed, a useful description is expected, and relevant tickets should be mentioned -->
+### Description: 
+Summarize the changes you're submitting in a few sentences, including Jira ticket ATL-xxxx if applicable.
 
-Fixes ATL-xxxx
+### Context: 
+It creates traceability on the matter tackled by the PR. It will also help the reviewer understand what the PR is about ---- linking to related issues, bug reports, or discussions for context.
```

Comment:
```suggestion
It creates traceability on the matter tackled by the PR. It will also help the reviewer understand the PR, linking to related issues, bug reports, or discussions for context.
```